sam-react-app/coreconcept/src/counter/Counter.test.tsx

```diff
@@ -0,0 +1,29 @@
+import React from 'react';
+import { render, fireEvent } from '@testing-library/react';
+import Counter from "./Counter";
+
+describe('CounterTest', () => {
```

Comment:
Please create a separate PR for tests so we can have one PR for each chapter.

Reply:
Ack